packages/backend/API.md

```diff
@@ -34,6 +34,16 @@ import { Stack } from 'aws-cdk-lib';
 
 export { a }
 
+// @public (undocumented)
+export type AmplifyStackBase = {
+    readonly stack: Stack;
+};
+
+// @public (undocumented)
+export type AmplifyStackResources = AmplifyStackBase & {
+    [K in keyof DefineStackProps]: Omit<ReturnType<DefineStackProps[K]['getInstance']>, keyof ResourceAccessAcceptorFactory>;
+};

```

Comment:
Seems like Typescript can't calculate this type (maybe because it is deeply nested with generic types?) fully so only `backend.stack1.resources.stack` is available in intellisense.